Makefile

```diff
@@ -132,6 +132,22 @@ $(DOCSDIR)/preview.svg: $(DOCSDIR)/preview.tex $(OTF)
 		$< 1>/dev/null || (cat $(basename $<).log && false)
 	mutool draw -q -r 200 -o $@ $(basename $@).pdf
 
+define unicode_coverage_table =
+	$(shell echo ttx -t cmap -o - $1.otf)
```

Comment:
What is the `ttx` call for?

Reply:
It's not yet, it's just one alternative for collecting data to actually generate the table with. This is a WIP ;-) I'm actually leaning towards `pyfontaine` instead to collect the same data, but for that I'm kind of waiting for [this upstream bug fix to land](https://github.com/googlefonts/pyfontaine/issues/110).